packages/infra/config/env-config.ts

```diff
@@ -98,8 +98,11 @@ export type EnvConfig = {
   iheGateway?: {
     vpcId: string;
     certArn: string;
+    ownershipVerificationCertArn: string;
     subdomain: string; // Subdomain for IHE integrations
     snsTopicArn?: string;
+    trustStoreBucketName: string;
+    trustStoreKey: string;
```

Comment:
the internal PR will need to be merged/approved before we can merge this one: https://github.com/metriport/metriport-internal/pull/1435/files

please update the release plan accordingly